mozperftest_tools/setup.py

```diff
@@ -0,0 +1,36 @@
+import setuptools
+
+PACKAGE_NAME = "mozperftest_tools"
+PACKAGE_VERSION = "0.1.0.dev9"
+
+# dependencies
+deps = ["numpy >=1.20.2", "matplotlib >=3.4.1,<3.5", "opencv-python", "scipy", "requests"]
```

Comment:
I hit some issue with these dependencies. I suggest removing the version requirements for now.